src/log.js

```diff
@@ -0,0 +1,11 @@
+const spikeLog = (str) => {
+  fs = require("fs");
+  const FILENAME = "../spikeLog.txt";
+
+  const time = new Date();
+  fs.appendFile(FILENAME, `${time} ~~ ${str}\n`, (err, t) => {
+    if (err) return console.log(err);
+  });
+};
+
+module.exports = { spikeLog };
```

Comment:
- There should be some way of specifying that a message is an error to make them stand out in the log, and potentially also in a separate error log as well.
- There is no concept of log rotation, so the log file will just continue to grow out of control.